code/modules/mob/logout.dm

```diff
@@ -27,6 +27,9 @@
 
 
 				send2irc("Server", "[cheesy_message]")
+	
+	if(client)
+		qdel(client)

```

Comment:
IIRC this will always be null, but i'll assume you've verified other wise?

also, I'd like to get away from putting anything relating to a client (dis)connecting in mob login/logout (in fact some of this should be moved to client new/del)